packages/compass-components/src/components/modal.tsx

```diff
@@ -1,21 +1,48 @@
 import React, { useEffect } from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { spacing } from '@leafygreen-ui/tokens';
 import { createLoggerAndTelemetry } from '@mongodb-js/compass-logging';
 const { track } = createLoggerAndTelemetry('COMPASS-UI');
 
 import { Modal as LeafyGreenModal } from './leafygreen';
 
+const variantStyles = {
+  'with-footer': css({
+    width: '600px',
```

Comment:
width, padding and letterSpacing are the same, maybe just `const baseStyles = {padding: 0, letterSpacing ...}` and then `{...baseStyles, paddingBottom}`?

Do we need these variants? we should have alternatives (playing around with removing the padding top from the footer for example), but the most direct question is, isn't this needed only for Info modals? we could put that only there and avoid variants for the content.

Reply:
I was trying to avoid having another modal class just for one line of css.